ctapipe/io/magiceventsource.py

```diff
@@ -0,0 +1,1352 @@
+
+
+import glob
+import re
+
+import h5py
```

Comment:
this adds another dependency of H5py which is not used in ctapipe so far (we use pytables for HDF5 access, and I would like to keep the required dependencies to a minimum).  Can you make that optional as with uproot?

Reply:
As the reading using uproot works well, do you need both event sources?

Reply:
h5py is by default anaconda and it is already in the evironment.yml in the master:
https://github.com/cta-observatory/ctapipe/blob/e7152482461d1f386f3a998e6ac567e21bcb93f7/environment.yml#L14